iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/KubernetesVerifier.java

```diff
@@ -76,7 +77,7 @@
 public class KubernetesVerifier {
 
   private static final Logger LOG = LoggerFactory.getLogger(KubernetesVerifier.class);
-  public static final Path BASE_DIR = Paths.get("src", "test", "resources", "checks");
+  public static final Path BASE_DIR = Paths.get("build", "resources", "test", "checks");
```

Comment:
This way we can create/modify files in a k8s/helm project, e.g. substitute placeholders with different namespaces, without messing the actual sources/resources

Reply:
Good idea. I thought about something similar in my PR but using temp dir.
Please verify if it has an influence on the Gradle cache (because some files will be created in the `build` dir).

Reply:
I didn't go with the temp dir approach because it would mean additional copying, and we don't modify most part of the resources.
For the Gradle cache, it probably is affected, but we should rerun tests if inputs change anyway. Ideally, these files should be cleaned up after test execution, but I'm not sure what would be the best approach, as KubernetesVerifier is not part of the Junit's lifecycle

Reply:
I added something with a callback; it should be added to each test class that uses this functionality, but at least it's a solution